ocs_ci/deployment/deployment.py

```
@@ -3747,6 +3748,388 @@ def _patch_ramen_csv(self, csv_prefix, namespace, new_image):
+            run_cmd(init_cmd, timeout=60)
+            logger.info("[ramenctl-e2e] Generated default config with ramenctl init")
+        except CommandFailed as e:
+            logger.error(f"[ramenctl-e2e] Failed to run ramenctl init: {e}")
+            return None
+
+        config_file = f"{ramenctl_dir}/config.yaml"
+
+        logger.info("[ramenctl-e2e] Building config content with cluster details")
+        try:
+            ramenctl_config = self._build_ramenctl_config_content(ramenctl_dir)
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)